realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -281,6 +283,35 @@ public void refresh() {
     }
 
     /**
+     * Blocks the current thread util transactions are committed by other threads to this Realm or
+     * {@link #stopWaitForChange()} is called from other threads.
+     *
+     * @return {@code true} if transactions are committed to this Realm, {@code false} if
```

Comment:
return true if a transaction has been committed and this instance has been refreshed, false if wait has been cancelled.